core/templates/components/common-layout-directives/common-elements/attribution-guide.component.ts

```diff
@@ -16,11 +16,12 @@
  * @fileoverview Component for the attribution guide.
  */
 
+import { BrowserCheckerService } from
+  'domain/utilities/browser-checker.service';
+
 import { Component, OnInit } from '@angular/core';
 import { downgradeComponent } from '@angular/upgrade/static';
 
-import { BrowserCheckerService } from
-  'domain/utilities/browser-checker.service';
 import { UrlService } from 'services/contextual/url.service';
```

Comment:
Why is this kept here?

Reply:
@vojtechjelinek All of these are grouped in this order `["builtin", "external", "parent", "sibling", "index"]`. You can see the configuration here (https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/order.md)

Reply:
Sure but then it seems that stuff from `domain` folder is evaluated as builtin which is wrong on the other note stuff from `services` is evaluated as internal (I guess).

Reply:
These link might be useful https://github.com/benmosher/eslint-plugin-import/blob/227d9a25098b359c11cce3c95d6168acdb87eb99/README.md#importinternal-regex
https://github.com/benmosher/eslint-plugin-import/blob/227d9a25098b359c11cce3c95d6168acdb87eb99/README.md#importexternal-module-folders